docs/content/release-process.md

```diff
@@ -80,6 +80,11 @@ To cut a release of Spin, you will need to do the following:
    `--certificate-identity` value should match this release, e.g.
    `https://github.com/fermyon/spin/.github/workflows/release.yml@refs/tags/v1.1.0`.
 
+1. Update the Spin [Homebrew
```

Comment:
Should we mention the specific mechanism envisioned for updating the formula?  eg via PR or...?